Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/survey/server/SurveyApiV1.kt

```diff
@@ -0,0 +1,18 @@
+package de.rki.coronawarnapp.datadonation.survey.server
+
+import com.google.gson.annotations.SerializedName
+import de.rki.coronawarnapp.server.protocols.internal.ppdd.EdusOtpRequestAndroid
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+interface SurveyApiV1 {
+
+    data class DataDonationResponse(
+        @SerializedName("errorCode") val errorCode: String?
```

Comment:
Shouldn't this contain the expiration date?

Reply:
I saw in other comments you said the expirationDate is obsolete, I didn't understand it, can you explain this?

Reply:
the spec was updated - there is no expiration date anymore

Reply:
https://github.com/corona-warn-app/cwa-app-tech-spec/pull/19/commits/bbb1ce4e818f925416d147a90474d56e11b7c206